activities/config_hinks.json

```diff
@@ -0,0 +1,16 @@
+{
+  "tasks": [
+      {
+      "layers": [
+        [
+          "d.rast", "map=flow"
+        ]
+      ],
```

Comment:
Looks like ponds are created but not added here to be visualized?

Reply:
Thank you, Anna! I added ponds to the list of layers with "map=ponds".